network/junos/junos_command.py

```diff
@@ -20,15 +20,15 @@
 ---
 module: junos_command
 version_added: "2.1"
-author: "Peter sprygada (@privateip)"
-short_description: Execute arbitrary commands on Juniper JUNOS devices
+author: "Peter Sprygada (@privateip)"
+short_description: Run a list of commands on Juniper JUNOS devices
 description:
   - Netork devices running Juniper JUNOS can execute a variety of commands
     to display both configuration and state data.  This module will take
     an ordered set of commands and execute them in JUNOS and return the
-    command results.  It also supports adding conditionals the
-    argument set to force the module to meet a set of criteria before
-    proceeding.
+    command results.  It also supports adding a set of conditionals waitfor
+    statemetns that are evaluated and must be true before the task is
```

Comment:
"statements" "considered"
There's something not quite right about the wording, too. Should it be "a set of conditional waitfor statements"?